src/renderer/components/helpers/ChatMethods.ts

```diff
@@ -206,6 +206,49 @@ export async function joinCall(
   }
 }
 
+/**
+ * Creates a new chat with given email address and returns chat id.
+ *
+ * In case an unknown email address was chosen or a chat does not exist yet the
+ * user will be prompted with a confirmation dialogue. In case the user aborts the
+ * action `null` is returned.
+ */
+export async function createChatByEmail(email: string): Promise<number | null> {
+  const accountId = selectedAccountId()
+
+  let contactId = await BackendRemote.rpc.lookupContactIdByAddr(
+    accountId,
+    email
+  )
+
+  const chatId = contactId
+    ? await BackendRemote.rpc.getChatIdByContactId(accountId, contactId)
```

Comment:
The `getChatIdByContactId` method was newly added to the JSON RPC exports and is not released yet, this is potentially a blocker